tests/scripts/thread-cert/node.py

```diff
@@ -2054,9 +2093,12 @@ def __init__(self, nodeid, name=None, **kwargs):
         self.name = name or ('Host%d' % nodeid)
         super().__init__(nodeid, **kwargs)
 
-    def start(self):
+    def start(self, start_radvd=True, prefix=config.DOMAIN_PREFIX, slaac=False):
```

Comment:
```suggestion
    def start(self, radvd_advertise_prefix=(config.DOMAIN_PREFIX, False)):
```
nit: merge to one argument?

Reply:
but reader loses the information that `False` stands for `slaac`.